src/tests/subscription.test.js

```diff
@@ -1,5 +1,63 @@
-// Basic test to avoid conflicts
-// REMOVE THIS TEST AND MAKE YOURS!
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import subscriptionsSeed from '../seeds/subscription';
+
+const app = require('../app');
+const Subscription = require('../models/Subscription');
+
+const mockSubscription = {
+  classId: '64667748fc13ae7f027543cc',
+  members: '6465537202739f6df0878cd1',
+  date: new Date('2022-10-29'),
+};
+
+beforeAll(async () => {
+  await Subscription.collection.insertMany(subscriptionsSeed);
+});
+
+describe('POST /subscription', () => {
+  test('should create a new subscription', async () => {
+    const response = await request(app).post('/api/subscription').send(mockSubscription);
+
+    expect(response.status).toBe(201);
+    expect(response.error).toBeFalsy();
+  });
+});
```

Comment:
Missing post test with code status 400 and 500

Reply:
Thank you for, i will add it